docs/geometry.rst

```diff
@@ -235,3 +235,27 @@ Each module is further subdivided into 8 sensor tiles.
    .. automethod:: inspect
 
    .. automethod:: compare
+
+.. _det-PNCCD:
+
+PNCCD
+-----
+
+PNCCD detectors have two movable modules of 1024×512 pixels, each of which is a
+single tile.
+
+.. autoclass:: PNCCDGeometry
+
+   .. automethod:: from_relative_positions
```

Comment:
```suggestion
.. autoclass:: PNCCDGeometry

   .. autoattribute:: expected_data_shape

   .. automethod:: from_relative_positions
```

Since `expected_data_shape` is mentioned in the class' docstring, it might be a good idea to spell it out. It also seems I forgot to add the `:attr:` tag to said property in that docstring.

Reply:
Gotcha, added in 3aa9613.